app/app.go

```diff
@@ -471,6 +471,10 @@ func MakeCodec() *codec.Codec {
 	return cdc
 }
 
+func (app *HeimdallApp) GetCaller() helper.ContractCaller {
```

Comment:
Is this really required,
why can't we directly use `app.caller`?

Reply:
the field is private to the struct, thus we cannot access it in the tests as they are running in `bor_test` package